src/NativeScript/JSErrors.mm

```diff
@@ -63,7 +63,7 @@ void reportFatalErrorBeforeShutdown(ExecState* execState, Exception* exception,
         std::cerr << "Native stack trace:\n";
         WTFReportBacktrace();
 
-        std::cerr << "JavaScript stack trace:\n";
```

Comment:
We'd better use `NSLog` instead of `std::cerr` because right now stderr lines are not added to the device logs.

Reply:
I tend to agree with that but we need to see if we want to swap all `std:cerr` calls with NSLog.